corehq/apps/aggregate_ucrs/models.py

```diff
@@ -136,6 +136,10 @@ def pk_columns(self):
                 columns.append(column_name)
         return columns
 
+    @memoized
+    def get_secondary_tables(self):
```

Comment:
nit: I don't love this method name, as it doesn't imply anything about why it might be different than `.secondary_tables`. I'd personally favor `cached_secondary_tables` or `memoized_secondary_tables` to be more explicit.

Reply:
i wrote the above on the first commit, but I think the name should still change even more after changing the return type. e.g. `get_(cached)_secondary_tables_with_adapters` or something. I'm wishy-washy on whether I think it's better to explicitly include the fact that it's cached or not since that's certainly not a convention we employ elsewhere in HQ.